.github/workflows/release-to-ngc.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish release images to NGC
+
+on:
+  workflow_dispatch:
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    
+    - name: Login to DockerHub
+      uses: docker/login-action@v3
+      with:
+          username: ${{ secrets.GPUCIBOT_DOCKERHUB_USER }}
+          password: ${{ secrets.GPUCIBOT_DOCKERHUB_TOKEN }}
+
+    - name: Login to NGC
+      uses: docker/login-action@v3
+      with:
+        registry: nvcr.io
+        username: ${{ secrets.NGC_DOCKER_USER }}
+        password: ${{ secrets.NGC_DOCKER_PASSWORD }}
+
+    - name: Run With DockerHub Token
+      run: |
+        #!/usr/bin/env bash
+        set -e
+        
+        for CUDA in 11.2 11.8 12.0; do
+            for PY in 3.9 3.10; do
```

Comment:
We should figure out a way to reuse the values in [matrix.yaml](https://github.com/rapidsai/docker/blob/branch-23.12/matrix.yaml) so that we're not redeclaring those values. Otherwise this will become a pain to maintain.

It might be worth exploring converting this into a matrix job.